docs/release_notes/release_notes-0.16.0.rst

```diff
@@ -73,6 +73,7 @@ Key Features and Updates
   * DOCS-#4628: Add to_parquet partial support notes (#4648)
   * DOCS-#4668: Set light theme for readthedocs page, remove theme switcher (#4669)
   * DOCS-#4748: Apply the Triage label to new issues.   
```

Comment:
Do you want to go ahead and add the PR number that was forgotten in the last PR?

Reply:
I don't think it would make sense to have another issue and separate PR for such a change.

Reply:
Done.